packages/mobile/src/app/UpgradeScreen.tsx

```diff
@@ -2,21 +2,24 @@ import FullscreenCTA from '@celo/react-components/components/FullscreenCTA'
 import * as React from 'react'
 import { WithTranslation } from 'react-i18next'
 import { Namespaces, withTranslation } from 'src/i18n'
-import { navigateToWalletPlayStorePage } from 'src/utils/linking'
+import { headerWithCloseButton } from 'src/navigator/Headers.v2'
+import { navigateToWalletStorePage } from 'src/utils/linking'
 
 type Props = WithTranslation
 
 class UpgradeScreen extends React.Component<Props> {
-  static navigationOptions = { header: null }
+  static navigationOptions = {
+    ...headerWithCloseButton,
+  }
```

Comment:
Looks like we didn't want this screen to be closed.
Sure we want to change this now?

Reply:
Well, otherwise there is no way to close this screen if you are not in the mood of updating atm